setup/helpers/perf_helper.rb

```diff
@@ -63,171 +73,192 @@ def setup_r10k
     end
   end
 
+  # rubocop:disable Security/Open
   def download_file(url, destination)
     raise "Specified URL does not exist: #{url}" unless link_exists?(url)
+
     puts "Downloading #{url} to #{destination}"
-    download = open(url)
-    IO.copy_stream(download, destination)
-    raise "Download was not successful" unless File.exists?(destination)
+    IO.copy_stream(open(URI(url)), destination)
+    raise 'Download was not successful' unless File.exist?(destination)
   end
+  # rubocop:enable Security/Open
 
   def perf_install_pe
-    if !is_pre_aio_version?
-      # Must include the dashboard so that split installs add these answers to classification
+    unless is_pre_aio_version?
+      # Must include the dashboard so that split installs add these answers
+      # to classification
       r10k_remote = '/opt/puppetlabs/server/data/puppetserver/r10k/control-repo'
       @options[:answers] ||= {}
-      @options[:answers]['puppet_enterprise::profile::master::r10k_remote'] = r10k_remote
-      @options[:answers]['puppet_enterprise::profile::puppetdb::node_ttl'] = '0s'
+      @options[:answers][
+        'puppet_enterprise::profile::master::r10k_remote'] = r10k_remote
+      @options[:answers][
+        'puppet_enterprise::profile::puppetdb::node_ttl'] = '0s'
     end
 
     step 'install PE' do
       install_lei
     end
   end
 
-  def has_cent7_repo?(package, version)
-    cent7_uri = URI("#{BASE_URL}/#{package}/#{version}/repo_configs/rpm/pl-#{package}-#{version}-el-7-x86_64.repo")
+  def cent7_repo?(package, version)
+    cent7_uri = URI([BASE_URL,
+                     package,
+                     version,
+                     'repo_configs',
+                     'rpm',
+                     "pl-#{package}-#{version}-el-7-x86_64.repo"].join('/'))
 
     response_code = Net::HTTP.start(cent7_uri.host, cent7_uri.port) do |http|
       http.head(cent7_uri.path).code
     end
 
-    if response_code != "200"
-      Beaker::Log.notify("Skipping #{package} version #{version} because it doesn't appear to have a cent7 repo")
+    if response_code != '200'
```

Comment:
It looks like double-quotes are being replaced with single-quotes in some places but not others. What is the rule / pattern here?